src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,23 @@
+import { deleteComment } from "../api";
+import { useEffect } from "react";
+
+function DeleteComment({ comment, setComments, setCommentCount }) {
+  const handleClick = () => {
+    deleteComment(comment.comment_id).then(() => {
```

Comment:
What would happen if we tried to delete while offline? Would the user receive any feedback? 🙂